ODIN_II/regression_test/benchmark/verilog/keywords/always/always_or_event_output

```diff
@@ -1,6 +1,6 @@
-out
-x
-x
-0
-0
-1
\ No newline at end of file
+out1 out2
```

Comment:
can you have a bit longer?? maybe at least 10-15 lines to really see different clock rates